syntax = "proto3";

message PeerUpdate {
    message PeerInfo {
        string name = 1;
        string host = 2;
        uint32 port = 3;
    }

    repeated PeerInfo add_peers = 1;
}